AdobeStockImageAdminUi/view/adminhtml/ui_component/adobe_stock_images_listing.xml

```diff
@@ -21,6 +21,7 @@
     <dataSource name="adobe_stock_images_listing_data_source" component="Magento_Ui/js/grid/provider">
         <settings>
             <storageConfig>
+                <component>Magento_AdobeStockImageAdminUi/js/grid/data-storage</component>
```

Comment:
Could you please fix static test with correct xml ?
Element 'component': This element is not expected. Expected is one of ( provider, namespace, path, param ).

Reply:
Sorry for this simple mistake. I saw that the component node is not included in the schema definition of the xml but I didn't get any error in the local because of the production mode but now I've fixed this issue.
Thanks